apisix/plugins/authz-keycloak.lua

```diff
@@ -114,6 +114,11 @@ local _M = {
 
 
 function _M.check_schema(conf)
+    local check = {"discovery", "token_endpoint", "resource_registration_endpoint",
+                    "access_denied_redirect_uri"}
+    core.utils.check_https(check, conf, plugin_name)
+    core.utils.check_tls_bool({conf.ssl_verify}, {"ssl_verify"}, plugin_name)
```

Comment:
I mean `check` and `conf` seem to be reversed. And it seems that `check_tls_bool` can continue to use the string pattern in `check_https` to get data in conf instead of supplying it upfront in the code.

Reply:
great catch 🙏🏼 , thanks for your detailed attention.